client/my-sites/plugins/minimum-jetpack-version-notice.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { warningNotice } from 'state/notices/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+import {
+	isJetpackSite,
+	siteHasMinimumJetpackVersion,
+	getSiteAdminUrl,
+	getSiteDomain
+} from 'state/sites/selectors';
+
+class MinimumJetpackVersionNotice extends Component {
```

Comment:
Minor one: the component name seems a little confusing to me. It would be more clear if it's something like `NonSupportedJetpackVersionNotice`, because the `MinimumJetpackVersionNotice` does not actually describe what the real problem is (or it does not describe that we're having a problem).